libs/external-router/driver/testing/src/config.ts

```diff
@@ -0,0 +1,10 @@
+import { InjectionToken } from '@angular/core';
+
+export type DaffExternalRouterType = string;
+
+export interface DaffExternalRouterDriverTestingConfig {
+	[url: string]: DaffExternalRouterType;
+}
+export const DAFF_EXTERNAL_ROUTER_DRIVER_TESTING_CONFIG = new InjectionToken<
```

Comment:
The `DaffExternalRouterDriverTestingConfig` needs a doc

Reply:
https://github.com/graycoreio/daffodil/pull/1240/commits/e84f901398d30c4af371fb257714725339f3640a